### DISCUSSION TITLE
请问官方经济白皮书可以在哪看到？

### DISCUSSION THREAD

**Discussion by 0xChelsea33** [filecoin-project/community-china#100] [新手上路]
请问官方经济白皮书可以在哪看到？

**Comment by maxvint** [comment]
@HinadaC 
- [Filecoin经济模型白皮书中文版](https://filecoin.io/zh-cn/2020-engineering-filecoins-economy-zh-cn.pdf)
- [Filecoin经济模型白皮书英文版](https://filecoin.io/2020-engineering-filecoins-economy-en.pdf)
- [Filecoin的流通供应机制](https://filecoin.io/zh-cn/blog/posts/%E4%BA%86%E8%A7%A3filecoin%E6%B5%81%E9%80%9A%E4%BE%9B%E5%BA%94%E6%9C%BA%E5%88%B6/)

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308,另接代封装业务，矿池搭建和运维服务，价格美丽，提供教学指导
